tests/integration/components/lt-body-test.js

```diff
@@ -33,7 +33,7 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let row = this.$('tr:not(.lt-scaffolding-row):first');
```

Comment:
@buschtoens I dont think this is needed since you have specific tests for when it is enabled. I prefer it like that since it is more contained. If we enabled it by default in the future, we can just turn it off for these test cases.